src/pages/PeopleTabPage.tsx

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from 'react';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { getPeople } from '../api';
+import { Person } from '../types';
+
+export const PeopleTabPage: React.FC = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[]>([]);
+  const [isLoadin, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(false);
```

Comment:
The variable `errorMessage` is initialized with a boolean value but used to store an error message. It can lead to confusion. Consider renaming it to `hasError` or `errorOccurred`, because now it seems that the error text should be stored there.